### PR TITLE
Added home screen

### PR DESCRIPTION
**Pull request by MarioZeldaDonkeyKong**
Startup progress working on screen #1

### CODE REVIEW

File: FSBeheer/FSBeheer/View/Home.xaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿<Window x:Class="FSBeheer.Home"
+﻿<Window x:Class="FSBeheer.View.Home"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        xmlns:local="clr-namespace:FSBeheer"
+        xmlns:local="clr-namespace:FSBeheer.View"
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/View/Home.xaml:6]: Why did you do this?
```suggestion
        xmlns:local="clr-namespace:FSBeheer.ViewD"
```

**Review by Evert-Arends — CHANGES_REQUESTED** [review]
Typo, please check and review.

File: FSBeheer/FSBeheer/View/Home.xaml (hunk 1 above)
Comment (MarioZeldaDonkeyKong) [reply FSBeheer/FSBeheer/View/Home.xaml:6]:
```suggestion
        xmlns:local="clr-namespace:FSBeheer.View"
```
Ja, this is better

Comment (Evert-Arends) [reply FSBeheer/FSBeheer/View/Home.xaml:6]: Okay, noted.

**Review by Evert-Arends — APPROVED** [review]

**Comment by Evert-Arends** [comment]
Code checked, and  it was okay

**Pull request merged**